OpenRA.Mods.Common/Traits/Radar/JamsRadar.cs

```diff
@@ -18,6 +18,9 @@ public class JamsRadarInfo : TraitInfo<JamsRadar>
 	{
 		[Desc("Range for jamming.")]
 		public readonly WDist Range = WDist.Zero;
+
+		[Desc("What diplomatic stances are affected.")]
+		public readonly Stance JamsRadarStances = Stance.Ally | Stance.Neutral;
```

Comment:
Uhm, you did this wrong. Neutral | Enemy, not Ally | Neutral was my request (you can remove the YAML definition thenafter as well.)